mdw-hub/web/assets/asset.html

```diff
@@ -32,6 +32,9 @@
           </a>            
           <a href="" class="btn btn-default" ng-if="authUser.hasRole('Site Admin')" ng-click="deleteAsset()">
             <span class="glyphicon glyphicon-remove"></span> Delete
+          </a>  
+          <a ng-if="task" href="" class="btn btn-default" ng-if="authUser.hasRole('Site Admin')" ng-click="viewInstances()">

```

Comment:
**Site Admin** is not required to view task instances.  You don't have to check any particular role at this time.